### DISCUSSION TITLE
Download failed IBM video player

### DISCUSSION THREAD

**Discussion by rickdgray** [aclap-dev/video-downloadhelper#702] [Help and Q&A]
### Your question

The download fails when trying to download the commencement video seen [here](https://commencement.utexas.edu/content/webcasts). Let me know if you have any questions. Thanks for the help!

```
{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://commencement.utexas.edu/content/webcasts",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tcp @ 00000295d9202440] Failed to resolve hostname __host__: The name does not resolve for the supplied parameters\r\n[hls @ 00000295d916b840] Error when loading first segment 'https://__host__/sjc/omega/vod/us1-5018afa8-59a4-4c3a-8c31-5a4c2bd7f6cd/31752000/111809058480000/plain/rfc/2/chunk_0_fd57ccd45b.m4v'\r\n[in#0 @ 00000295d916b6c0] Error opening input: I/O error\r\nError opening input file https://a93e545d-lp-omega.ums.services.video.ibm.com/playlist/auditorium/recorded/133606079/2/playlist.m3u8?token=1208182_1716788893514&appType=7&appVersion=1&chunkingType=rfc&segmentType=mp4&content=video&sgn=475c83235cdfcff152e9df6877503fd24c6d8b55&cdnHost=__host__.\r\nError opening input files: I/O error\r\n",
    "id": "coapp_failure_2ba052be-f584-4e4b-80d8-64b5b0bc5ab9",
    "downloadable_id": "downloadable_810041213339881"
  }
}
```

### Addon & CoApp version and details

```
version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"8fa4d44454394645"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\rickd\\dwhelper",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 6
}
```

**Comment by paulrouget** [comment]
I can reproduce. That's an unexpected non-standard behavior. Will look at this!

**Reply by GrampaWildWilly** [reply]
Hmmm......  On the page mentioned in the initial post above, VDH doesn't find any usable media.  But there are links to a few years of commencements lower down on the page.  I picked the 2023 one:

https://vimeo.com/827380262

Clicking the link for the 2023 commencement took me to Vimeo.  That's a bit unexpected.  Anyway, I launched a VDH download of that.  It's a bit over 2 hours long & I'm getting only a little over 3 million bytes per second download speed.  So it's going to take a while to complete.  But it's downloading for me just fine.  I'm using Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.  So I'm surprised, Paul, that you can reproduce this.  Or is there something on the main page that isn't right?  I see a black box on the main page that eventually pops up a message offering to help me:

![image](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/0ae196e1-3728-4188-b214-a8a71219f234)

When I click that link, this is the help that is offered:

![image](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/afcb29ce-4d67-42e9-ba6e-378ddedf98d1)

Not too helpful.  I don't think I want to bother whoever is at the other end of that troubleshooting form.  Am I not seeing what you guys are seeing?

Anyway, I've babbled on long enough here that the download has completed.  This is what I've gotten:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/6aa50c02-519f-4cfe-bb78-f54f3fafef9f)

And it plays fine in VLC, although I've only skimmed it, sampling it at intervals.  It seems just fine.

I have to be missing something . . .

**Reply by rickdgray** [reply]
They seemed to have switched to a different provider. They used Vimeo previously but not this past year it seems.

**Reply by rickdgray** [reply]
Hi, just wanted to follow up on this. Has there been any movement?
